packages/framework/assets/styles/admin/todo.less

```diff
@@ -9,3 +9,8 @@
 input[name="product_form[imageGroup][images][uploadedFilenames][]"] {
 	display: none;
 }
+
+.select2 {
+  width: 100% !important;
```

Comment:
wrong indentation

Reply:
7959a92b1c33386469157d496f35f4ceafa44476

Reply:
closed - it will be resolved in responsive version https://github.com/shopsys/shopsys/pull/1806